components/access-management/Makefile

```diff
@@ -1,30 +1,31 @@
 REGISTRY_PROJECT ?= kubeflow-dev
-GOLANG_VERSION ?= 1.12
-GCLOUD_PROJECT ?= kubeflow-images-public
-IMG ?= gcr.io/$(GCLOUD_PROJECT)/kfam
 PROJECT ?= kubeflow-dev
-TAG ?= $(eval TAG := $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)-$(shell git diff | shasum -a256 | cut -c -6))$(TAG)
+GO_SWAGGER_URL ?= https://github.com/go-swagger/go-swagger/releases/download/v0.21.0/swagger_linux_amd64
 
-CHANGED_FILES := $(shell git diff-files --relative=components/centraldashboard)
+GIT_VERSION := $(shell git describe --tags --always --dirty)
 
-ifeq ($(strip $(CHANGED_FILES)),)
-# Changed files is empty; not dirty
-# Don't include --dirty because it could be dirty if files outside the ones we care
-# about changed.
-GIT_VERSION := $(shell git describe --tags --long)
-else
-GIT_VERSION := $(shell git describe --tags --long)-dirty-$(shell git diff | shasum -a256 | cut -c -6)
-endif
+IMG ?= kfam
+TAG ?= $(GIT_VERSION)
 
-build:
+generate-go-client: bin/swagger
+	bin/swagger generate client -f api/swagger.yaml -t api/go_client
+
+bin/swagger:
+	mkdir -p bin
+	wget -O bin/swagger '$(GO_SWAGGER_URL)'
+	chmod +x bin/swagger
+
+docker-build:
 	docker build -t $(IMG):$(TAG) .
 
+docker-push: 
+	docker push $(IMG):$(TAG)
+
+image: docker-build docker-push 
+
 build-gcb:
```

Comment:
Can you also remove this rule? We don't use `build-gcb` anywhere in the project.

It's a good chance to clean up the project

Reply:
Done!